UI/Web/src/app/pdf-reader/_components/pdf-reader/pdf-reader.component.html

```diff
@@ -2,23 +2,17 @@
   @if (accountService.currentUser$ | async; as user) {
     <div class="{{theme}}" #container>
 
-      @if (isLoading) {
-        <div class="loading mx-auto" style="min-width: 200px; width: 600px;">
-          <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
-          {{t('loading-message')}}
+      <div class="progress-container row g-0 align-items-center">
+        <div class="progress" style="height: 5px;">
+          <div class="progress-bar" role="progressbar" [ngStyle]="{'width': loadPercent + '%'}" [attr.aria-valuenow]="loadPercent" aria-valuemin="0" aria-valuemax="100"></div>
```

Comment:
The problem with this is it leaves the progress bar once it's at 100%. Let's remove the bar after loading is done and leave an `<span class="visually-hidden">{{t('loading-message')}}</span> here as well. 

Let's tweak the message in en.json to "loading..."